README.md

```diff
@@ -54,30 +129,60 @@ const credential = await navigator.credentials.get({
 });
 ```
 
-## Confirming payment
+## Authenticating a payment
 
-Any origin may invoke the [Payment Request API](https://w3c.github.io/payment-request/) to prompt the user to verify a credential created by any other origin. The `PaymentRequest.show()` method must require a user gesture and display user interface with the amount of payment and the hostname of the top-level context where the `PaymentRequest` API was invoked.
+Any origin may invoke the [Payment Request API](https://w3c.github.io/payment-request/) with the `secure-payment-confirmation` payment method to prompt the user to verify a SecurePaymentCredential created by any other origin. The `PaymentRequest.show()` method must require a user gesture. The browser will display a native user interface with the payment amount and the payee origin, which is taken to be the origin of the top-level context where the `PaymentRequest` API was invoked.
+
+Proposed new `secure-payment-confirmation` payment method:
+
+```webidl
+dictionary SecurePaymentConfirmationRequest {
+  SecurePaymentConfirmationAction action;
+  required DOMString instrumentId;
+  required BufferSource networkData;
```

Comment:
@btidor-stripe: Do you think the `networkData` needs to be specified as part of Secure Payment Confirmation or can we leave it opaque, and let the underlying payment rails (e.g. 3DS in the current use case) dictate what goes into it?

Since the browser will just be hashing this data and then getting the authenticator to sign over it, it keeps things simple if the detailed structure is left out of the spec. It also makes it more extensible to other payment rails in the future.

Reply:
Agreed! I don't think the browser needs to know anything about what goes into `networkData` -- my thought was that payment systems might suggest some or all of its contents, the same way they currently define standard cryptogram formats.